libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -588,6 +588,37 @@ bool AP_BattMonitor::reset_remaining_mask(uint16_t battery_mask, float percentag
     return ret;
 }
 
+// Logs smart battery messages
+void AP_BattMonitor::log_smart_battery_info() const
+{    
+#if HAL_SMART_BATT_LOG_ENABLED
+    AP_Logger *logger = AP_Logger::get_singleton();
+
+if (logger != nullptr && logger->should_log(_log_battery_bit)) {
```

Comment:
indentation

Reply:
this got removed to solve the problem of logging information in case of a new log file being opened